zerver/webhooks/sonarqube/doc.md

```diff
@@ -0,0 +1,15 @@
+Get Zulip notifications for your Sonarqube code analysis!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+   By default, the integration will use the project's name as the topic.
+
+1. On SonarQube Select **Administration** either in your project or on global level.
+   Select **Webhooks** (on global level **Configurations > Webhooks**). Click **Create**.
```

Comment:
I would rephrase this one slightly, like so:
1. To configure webhooks for a specific SonarQube project, go to the project and select **Administration**. Select **Webhooks** and click **Create**. **Note**: you can also configure webhooks globally by going to **Configurations -> Webhooks** in SonarQube.

Reply:
Ok